### ISSUE TITLE
error checkDebugDuplicateClasses work-runtime-2.8.0-runtime

### ISSUE THREAD

**Issue by rangga-synergy** [microsoft/clarity-apps#7]
Hi there, thank you for creating this library. 
But I got error after install `react-native-clarify` v0.0.4

```
Execution failed for task ':app:checkDebugDuplicateClasses'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.CheckDuplicatesRunnable
   > Duplicate class androidx.work.OneTimeWorkRequestKt found in modules work-runtime-2.8.0-runtime (androidx.work:work-runtime:2.8.0) and work-runtime-ktx-2.7.1-runtime (androidx.work:work-runtime-ktx:2.7.1)
     Duplicate class androidx.work.PeriodicWorkRequestKt found in modules work-runtime-2.8.0-runtime (androidx.work:work-runtime:2.8.0) and work-runtime-ktx-2.7.1-runtime (androidx.work:work-runtime-ktx:2.7.1)
```

This is my Android Gradle info:
  ```
        buildToolsVersion = "33.0.0"
        kotlinVersion = "1.7.0"
        minSdkVersion = 23
        compileSdkVersion = 33
        targetSdkVersion = 33

        // We use NDK 23 which has both M1 support and is the side-by-side NDK version from AGP.
        ndkVersion = "23.1.7779620"
```

And this is my React Native info:
```
"react-native": "0.71.6",
"react": "18.2.0"
```

Do you know what i have to do to fix this issue?

Anyway, let me know if u need more info from my side.

Thank you so much!

**Comment by gustavolsilvano** [comment]
Any update?

**Comment by nouranmaher86** [comment]
Hello-
We have released a new version of the SDK today 1.3.1 that includes a fix for this error. Please give it a try and let us know.
